docs/s3-supported-api.md

```diff
@@ -34,6 +34,13 @@
 | Abort Multipart Upload |
 | List Part |
 
+:page_with_curl: **Note:** CORTX has following limitations in Multipart upload API:
```

Comment:
You can change it to  
limitation in Multipart upload APIs:

Reply:
done